tutor/resources/styles/book-content/index.scss

```diff
@@ -37,6 +37,7 @@
   @extend %tutor-book-content-teacher;
   @extend %tutor-book-content-questions;
 
+  @include book-content-interactives();
   @include tutor-book-notes();
```

Comment:
heh, I bet this was one of those "takes forever to investigate but only a second to fix".   That must have been missed when I translated the less to scss.  Glad you figured it out